#!/usr/bin/env python
PACKAGE = "optar"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()


gen.add("estimate_distance_thresh_meters",		double_t,	0, "Maximu distance allowed between a new estimation and the previous one, in meters", 5, 0, 50)
gen.add("handler_no_msg_timeout_secs",		double_t,	0, "Time without any new message after which the handler for a specific phone is removed, in seconds",   5, 0, 60)
gen.add("startup_frames_num",				int_t,		0, "Number of frames to use to initialize the kalman filter",  4, 0, 30)

exit(gen.generate(PACKAGE, "optar", "OptarAggregatorParameters"))